docs/en/creating-widgets/supplemental.md

```diff
@@ -0,0 +1,972 @@
+| `Widget | constructor` | No                | Typically, components will pass in `Widget` as a generic type reference to an imported widget. Several types of `constructor`s can also be passed, allowing Dojo to instantiate widgets in a variety of different ways. These allow for advanced features such as deferred or lazy loading.                                                                                                                                                                                                                                                    |
+| `properties`           | `v`: Yes, `w`: No | The [set of properties used to configure the newly created VDOM node](#virtual-node-properties). These also allow the framework to determine whether the node has been updated and should therefore be re-rendered.                                                                                                                                                                                                                                                                                                                            |
+| `children`             | Yes               | An array of nodes to render as children of the newly created node. This can also include any text node children as literal strings, if required. Widgets typically encapsulate their own children, so this argument is more likely to be used with `v()` than `w()`.                                                                                                                                                                                                                                                                           |
+
+### Virtual nodes example
+
+The following sample widget includes a more typical render function that returns a `VNode`. It has an intended structural representation of a simple `div` DOM element which includes a text child node:
+
+> src/widgets/MyWidget.ts
+
+**Function-based variant:**
+
+```ts
+import { create, v } from '@dojo/framework/core/vdom';
+
+const factory = create();
+
+export default factory(function MyWidget() {
+	return v('div', ['Hello, Dojo!']);
+});
+```
+
+**Class-based variant:**
+
+```ts
+import WidgetBase from '@dojo/framework/core/WidgetBase';
+import { v } from '@dojo/framework/core/vdom';
+
+export default class MyWidget extends WidgetBase {
+	protected render() {
+		return v('div', ['Hello, Dojo!']);
+	}
+}
+```
+
+### Composition example
+
+Similarly, widgets can compose one another using the `w()` method, and also output several nodes of both types to form a more complex structural hierarchy:
+
+> src/widgets/MyComposingWidget.ts
+
+**Function-based variant:**
+
+```ts
+import { create, v, w } from '@dojo/framework/core/vdom';
+
+const factory = create();
+
+import MyWidget from './MyWidget';
+
+export default factory(function MyComposingWidget() =>
```

Comment:
This is not valid syntax with the `=>` - it will need to be 
```ts
export default factory(function MyComposingWidget() {
    return v('div', ['This widget outputs several virtual nodes in a hierarchy', w(MyWidget, {})]);
});
```

Reply:
Missed this one when converting over to named functions.